src/Microsoft.AspNetCore.Blazor.Templates/content/BlazorServerSide-CSharp/BlazorServerSide-CSharp.Client/WeatherForecast.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace BlazorServerSide_CSharp.Client
+{
+    public class WeatherForecast
+    {
+        public DateTime Date { get; set; }
+        public int TemperatureC { get; set; }
+        public int TemperatureF { get; set; }
+        public string Summary { get; set; }
+    }
+}
```

Comment:
It might be nice to have `WeatherForecast` and `WeatherForecastService` in a subdir called `Services`. What do you think?

Reply:
Sure